pkg/interactor/list_data_test.go

```diff
@@ -0,0 +1,140 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+
+	thingentities "github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+type ListDataTestCase struct {
+	name               string
+	authorization      string
+	query              *entities.Query
+	expectedData       []entities.Data
+	expectedDataError  error
+	expectedThing      []*thingentities.Thing
+	expectedThingError error
```

Comment:
Please, consider the same suggestions I did on `save_data_test.go`

Reply:
Done!